cmd/aro/mirror.go

```diff
@@ -156,6 +165,9 @@ func mirror(ctx context.Context, _log *logrus.Entry) error {
 		// https://quay.io/repository/app-sre/hive?tab=tags
 		"quay.io/app-sre/hive:8796c4f534",
 
+		// https://quay.io/repository/openshift/aro-must-gather?tab=tags
+		"quay.io/openshift/aro-must-gather:latest",
```

Comment:
I don't think this image exists. Can you pull it locally?

If you run `oc adm must-gather` locally you will see it pulls `registry.redhat.io/openshift4/ose-must-gather:latest` which I believe would be the correct one we'd want to mirror and use.